extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java

```diff
@@ -105,6 +107,10 @@ public ImportResult importItem(
       throws Exception {
     // Ensure credential is populated
     getOrCreateCredential(authData);
+    MicrosoftTransmogrificationConfig config = new MicrosoftTransmogrificationConfig();
+    resource.transmogrifyPhotos(config);
+    resource.transmogrifyAlbums(config);
```

Comment:
I might consider combining these two together in the transmogrification interface, but I'll defer to you on that

Reply:
good idea, i'll put in a resource.transmogrify()

Reply:
we should move it over to ContainerResource at some point